CONTRIBUTING.md

```diff
@@ -107,69 +108,18 @@ Once installed, Mongo Orchestration can be started with
-
 The test suite uses the ``MONGODB_URI`` environment variable as connection
-string to run all tests. In order to make the URI available to the test suite,
-you can run the following for a "deployment" in the *root* of the MongoDB
-Driver GIT checkout:
-
-```
-export TRAVIS_BUILD_DIR=`pwd`
-DEPLOYMENT=STANDALONE_AUTH .travis.scripts/setup_mo.sh
-export MONGODB_URI=`cat /tmp/uri.txt`
-```
+string to run all tests. If not set, tests assume MongoDB is listening on the
+default MongoDB port (27017) on localhost.
 
 With this set-up, the tests can be run with `make test`.
 
-### VM-based Mongo Orchestration (legacy set-up)
-
-Alternative to the Travis CI set-up, our test suite also includes scripts to configure test environments
-with [Vagrant](https://www.vagrantup.com/) and
-[Mongo Orchestration](https://github.com/10gen/mongo-orchestration).
-The deployments started in this Vagrant image have hard coded URLs to be used
-with the ``MONGODB_URI`` environment variable:
-
-Deployment                  | URI
---------------------------- | ---
-Standalone (MongoDB 4.0)    | `mongodb://192.168.112.10:2000`
-Standalone (MongoDB 3.0)    | `mongodb://192.168.112.10:2700`
-Standalone with SSL         | `mongodb://192.168.112.10:2100`
-Standalone with Auth        | `mongodb://root:toor@192.168.112.10:2200/?authSource=admin`
-Standalone with X509 Auth   | `mongodb://C=US,ST=New York,L=New York City,O=MongoDB,OU=KernelUser,CN=client@192.168.112.10:2300/?authSource=$external&authMechanism=MONGODB-X509`
-Standalone with Plain Auth  | `mongodb://root:toor@192.168.112.10:2400/?authSource=admin`
-Replicaset (MongoDB 4.0)    | `mongodb://192.168.112.10:3000,192.168.112.10:3001,192.168.112.10:3002/?replicaSet=REPLICASET`
-Replicaset (MongoDB 3.0)    | `mongodb://192.168.112.10:3100,192.168.112.10:3101,192.168.112.10:3102/?replicaSet=REPLICASET_30`
-Replicaset (MongoDB 3.6)    | `mongodb://192.168.112.10:3200,192.168.112.10:3201,192.168.112.10:3202/?replicaSet=REPLICASET_36`
-
-The Vagrant images can be started by using:
-
-```
-$ make vm             # Starts the test VMs with Vagrant
-$ make test-bootstrap # Starts the mongod servers within the test VM
-```
-
-After this set-up is completed, you need to export the `MONGODB_URI`
-environment variables with one of the values from the table above. The `test`
-make target may be used to execute the test suite:
-
-```
-$ make test # Executes the test suite against the VMs
-```
-
-To find out which VM servers are running at a later point in time, you can run
-`make test-bootstrap` to obtain a list of deployments and their URIs.
-
-#### Restarting Mongo Orchestration
-
-If something goes awry in the test VM, you can reload it by running:
+If Mongo Orchestration is running on non-default settings, the
+``MONGO_ORCHESTRATION_URI`` environment variable can be used to specify where
+mongo-orchestration is listening:
 
 ```
-make test-bootstrap
+MONGO_ORCHESTRATION_URI=http://some-host:1234/v1 make test
```

Comment:
nit: either use example.com (what it's intended for) for localhost. In practice, I can't imagine MO would run somewhere _other_ than localhost. This is generally just needed if the the port is changed. I think `localhost:1234` would be fine.